samples/bookinfo/kube/route-rule-all-v1.yaml

```diff
@@ -13,6 +14,105 @@ spec:
 apiVersion: config.istio.io/v1alpha2
 kind: RouteRule
 metadata:
+  name: ingress-deny-all
+spec:
+  destination:
+    name: productpage
+  match:
+    source:
+      labels:
+        istio: ingress
+  precedence: 2
+  route:
+  - weight: 100
+  httpFault:
+    abort:
+      percent: 100
+      httpStatus: 403
+---
+apiVersion: config.istio.io/v1alpha2
+kind: RouteRule
+metadata:
+  name: ingress-products-api
```

Comment:
Sorry I don't fully understand why we need 3 or 4 route rules for product page now.   Could u explain?   Otherwise LGTM

Reply:
https://istio.io/docs/tasks/traffic-management/ingress.html . We not using ingress spec any more. We need one route rule for each ingress path.

Reply:
Each route rule exposes a particular path in product page.. Everything else will be denied.

Reply:
Thanks!  Sorry I missed this change earlier... this is more complicated than I had thought.